vagrant/README.md

```diff
@@ -0,0 +1,138 @@
+Vagrant provider for jclouds
+============================
+
+Building
+--------
+
+1. Vagrant bindings
+  * `git clone https://github.com/neykov/vagrant-java-bindings`
+  * `cd vagrant-java-bindings`
+  * `mvn clean install`
+  * Copy `target/vagrant-java-bindings-0.0.1-SNAPSHOT.jar` to jcloud's classpath
+2. Vagrant provider
+  * `git clone https://github.com/neykov/jclouds-vagrant`
+  * `cd jclouds-vagrant/vagrant`
+  * `mvn clean install`
+  * Copy `target/vagrant-2.0.0-SNAPSHOT.jar` to jcloud's classpath
+
+Using in Apache Brooklyn
+------------------------
+
+Download a Vagrant box:
+```
+vagrant box add ubuntu/trustry64
```

Comment:
That's core `vagrant` functionality. If `box add` doesn't work, then there's something broken in your vagrant install.
Some things to try out:
* running it with `--debug` at the end - any giveaways there?
* does it fail with any box or just `ubuntu/trusty64`?
* what's in your `~/.vagrant.d/boxes` folder?